app/assets/stylesheets/gif.scss

```diff
@@ -115,6 +121,14 @@
   }
 }
 
+.gif-result-card {
+  border: none !important;
```

Comment:
this style could be pretty common, so I recommend to create 'border-none' class and use it on place of need.